src/fabric/src/fabric2_db.erl

```diff
@@ -1001,6 +1002,11 @@ maybe_set_user_ctx(Db, Options) ->
     end.
 
 
+set_tx_options(Db, Options) ->
```

Comment:
You've got the transaction options left in here but I don't see them being used anywhere. I'd rather not mix that in with this commit. And I'd probably argue against adding it until we actually need it if I'm not missing where its being used.

Reply:
It was mostly for tests to avoid setting up mocks. But good call on not mixing it together.